src/commcare_cloud/ansible/roles/bootstrap-users/tasks/main.yml

```
@@ -141,7 +141,7 @@
 - name: check users password valid time
   shell: getent shadow "{{ item }}" | cut -d':' -f5
   register: users_pw_valid
-  loop: "{{ dev_users.present }} + ['ansible', '{{ cchq_user }}']"
+  loop: "{{ ['ubuntu'] + ['ansible', cchq_user] }}"
```

This doesn't look like it does what it did before. Would this work instead?
```suggestion
  loop: "{{ dev_users.present + ['ansible', cchq_user] }}"
```